README.md

```diff
@@ -1,35 +1,59 @@
 ![](https://img.shields.io/badge/Certification%20Level-Community-28A745?link=https://github.com/cyberark/community/blob/master/Conjur/conventions/certification-levels.md)
 
-## CyberArk Ansible Conjur Collection
+# CyberArk Ansible Conjur Collection
 
-****
+This collection contains plugins to be used for CyberArk Conjur & DAP (Dynamic Access Provider) hosted in [ansible galaxy](https://galaxy.ansible.com/cyberark/conjur).
 
-### cyberark.conjur
+## Table of Contents
+- [CyberArk Ansible Conjur Collection](#cyberark-ansible-conjur-collection)
+  * [Requirements](#requirements)
+  * [conjur_variable Lookup Plugin](#conjur_variable-lookup-plugin)
+    + [Role Variables](#role-variables)
+    + [Dependencies](#dependencies)
+    + [Example Playbook](#example-playbook)
+  * [Conjur Ansible Role](#conjur-ansible-role)
+    + [Usage](#usage)
+    + [Role Variables](#role-variables-1)
+    + [Dependencies](#dependencies)
+    + [Example Playbook](#example-playbook-1)
+    + [Summon & Service Managers](#summon---service-managers)
+    + [Recommendations](#recommendations)
+  * [Contributing](#contributing)
+  * [License](#license)
 
-This collection contains plugins to be used for CyberArk Conjur & DAP hosted in [ansible galaxy](https://galaxy.ansible.com/cyberark/conjur).
+<!-- Table of contents generated with markdown-toc
+http://ecotrust-canada.github.io/markdown-toc/ -->
 
-### Requirements
+## Requirements
 
-- [CyberArk Conjur Open Source](https://www.conjur.org) v1.x+ or
-- [CyberArk Dynamic Access Provider](https://docs.cyberark.com/Product-Doc/OnlineHelp/AAM-DAP/Latest/en/Content/Resources/_TopNav/cc_Home.htm) v10.x+
-- Ansible v2.9+
+- conjur_variable Lookup Plugin
+  - [CyberArk Conjur Open Source](https://www.conjur.org) v1.x+ or
+  - [CyberArk Dynamic Access Provider](https://docs.cyberark.com/Product-Doc/OnlineHelp/AAM-DAP/Latest/en/Content/Resources/_TopNav/cc_Home.htm) v10.x+
+  - Ansible >= 2.9
 
-### Role Variables
-
-None.
-<br>
-<br>
+- Conjur Role
+  - A running Conjur service that is accessible from the target nodes.
+  - Ansible >= 2.3.0.0
```

Comment:
I'd suggest we go with Ansible >= 2.9 here, and make that the standard for the whole collection. that is, don't separate requirements into separate sections for the role / lookup plugin - just have one requirements section for the plugin, and use the one you have for the lookup plugin above.